<launch>
  <node pkg="match_claw_machine" type="joy_to_twist_bounded.py" name="joy_to_twist_bounded" output="screen">
    <param name="joy_topic"  value="/joy"/>
    <param name="pose_topic" value="/mur620b/UR10_r/ur_calibrated_pose"/>
    <param name="cmd_topic"  value="/mur620b/UR10_r/twist_controller/command_collision_free"/>

    <!-- Bounds -->
    <param name="x_min" value="0.30"/>
    <param name="x_max" value="0.90"/>
    <param name="y_min" value="-0.0"/>
    <param name="y_max" value="0.90"/>

    <!-- Controls -->
    <param name="axis_x" value="0"/>
    <param name="axis_y" value="1"/>
    <param name="enable_button" value="4"/>  <!-- LB z.B.; -1 = immer aktiv -->
    <param name="deadzone" value="0.08"/>

    <!-- Dynamics -->
    <param name="v_max" value="0.15"/>
    <param name="a_max" value="0.40"/>
    <param name="rate_hz" value="100.0"/>
  </node>
</launch>
